services/cd-service/pkg/service/git.go

```diff
@@ -226,28 +226,6 @@ func (s *GitServer) GetEvents(ctx context.Context, transaction *sql.Tx, fs billy
 	var result []*api.Event
 	parts := strings.Split(commitPath, "/")
 	commitID := parts[len(parts)-2] + parts[len(parts)-1]
-	allEventsPath := fs.Join(commitPath, "events")
-	potentialEventDirs, err := fs.ReadDir(allEventsPath)
-	if err != nil {
-		return nil, fmt.Errorf("could not read events directory '%s': %v", allEventsPath, err)
-	}
-	for i := range potentialEventDirs {
-		oneEventDir := potentialEventDirs[i]
-		if oneEventDir.IsDir() {
-			fileName := oneEventDir.Name()
-			rawUUID, err := timeuuid.ParseUUID(fileName)
```

Comment:
getEvents was fetching both from the manifest and the database. We now only fetch from the database